### PR TITLE
Dev#711

### PR DESCRIPTION
**Pull request by rudosch**
As suggested in issue #711 I'm proposing the following changes:
- fix minor spelling and grammar errors
- fix French translation file `/main/resources/i18n/text_fr.properties`
- add missing `response.setContentType` for some servlets
- fix some deprecation warnings, e.g. `new Integer()` -> `Integer.valueOf()`
- fix some html placeholders (`setup.jsp`) and select option (IDOR challenge 2)

### CODE REVIEW

**Comment by rudosch** [comment]
Sorry for the stupid syntax errors in initial `Getter.java` fixes ...

File: src/main/resources/i18n/challenges/injection/ac8f3f6224b1ea3fb8a0f017aadd0d84013ea2c80e232c980e54dd753700123e.properties [hunk 1]
```diff
@@ -1,5 +1,5 @@
 challenge.challengeName = XXE Injection 1
-challenge.description =  To complete this challenge you must read the contents JSON's file 
+challenge.description =  To complete this challenge you must read the contents of file
```
Comment (ismisepaul) [inline src/main/resources/i18n/challenges/injection/ac8f3f6224b1ea3fb8a0f017aadd0d84013ea2c80e232c980e54dd753700123e.properties]: Add `the` between `of file`

like so

`To complete this challenge you must read the contents of the file`

File: src/main/webapp/challenges/vc9b78627df2c032ceaf7375df1d847e47ed7abac2a4ce4cb6086646e0f313a4.jsp [hunk 2]
```diff
@@ -82,8 +82,8 @@
 					<tr>
 						</td>
 						<select id='userId' style='width: 300px;' multiple>
-							<option value="c81e728d9d4c2f636f067f89cc14862c">Paul
-								Bourke</option>
+							<option value="c81e728d9d4c2f636f067f89cc14862c">Joe
+								Sullivan</option>
```
Comment (ismisepaul) [inline src/main/webapp/challenges/vc9b78627df2c032ceaf7375df1d847e47ed7abac2a4ce4cb6086646e0f313a4.jsp:86]: Not sure if this change is necessary?

**Review by ismisepaul — COMMENTED** [review]
Thanks @rudosch this is a nice addition! 

I've two comments if you could take a look before we merge this in.

File: src/main/resources/i18n/challenges/injection/ac8f3f6224b1ea3fb8a0f017aadd0d84013ea2c80e232c980e54dd753700123e.properties (hunk 1 above)
Comment (rudosch) [reply src/main/resources/i18n/challenges/injection/ac8f3f6224b1ea3fb8a0f017aadd0d84013ea2c80e232c980e54dd753700123e.properties]: OK, I will change this.

File: src/main/webapp/challenges/vc9b78627df2c032ceaf7375df1d847e47ed7abac2a4ce4cb6086646e0f313a4.jsp (hunk 2 above)
Comment (rudosch) [reply src/main/webapp/challenges/vc9b78627df2c032ceaf7375df1d847e47ed7abac2a4ce4cb6086646e0f313a4.jsp:86]: The userid `c81e728d9d4c2f636f067f89cc14862c` belongs to Joe Sullivan, not Paul Burke. It's just a small logical error, nothing really important. Nevertheless, I propose to correct this.

Comment (ismisepaul) [reply src/main/webapp/challenges/vc9b78627df2c032ceaf7375df1d847e47ed7abac2a4ce4cb6086646e0f313a4.jsp:86]: Makes sense now, thanks.

**Comment by rudosch** [comment]
Since I am currently using SecurityShepherd in class, I may have some more suggestions for corrections. So if you don't mind, I'll probably do some more PRs soon ...

**Review by ismisepaul — APPROVED** [review]
LGTM

**Comment by ismisepaul** [comment]
@rudosch that's great! Feel free to do as many as you like. We'll get them looked at and merged in

**Pull request merged**